### PR TITLE
Removed mentions of Java 8

### PR DESCRIPTION
**Pull request by CharlesDuboisSAP**
## What Has Changed?

I updated all mentions of Java 8 to Java 17 in the v5 docs
This is part of [Update Major Dependency Versions#351](https://github.com/SAP/cloud-sdk-java-backlog/issues/351)
Note to reviewer: Double check the changes in the linux guide

### CODE REVIEW

File: docs-java_versioned_docs/version-v5/getting-started.mdx [hunk 1]
```diff
@@ -26,102 +26,23 @@ To get started with the SAP Cloud SDK for Java you can either create a new proje
...
-[INFO] ------------------------------------------------------------------------
-```
-
-</TabItem>
-<TabItem value="spring6">
-
 ```bash
 mvn archetype:generate "-DarchetypeGroupId=com.sap.cloud.sdk.archetypes" \
-    "-DarchetypeArtifactId=scp-cf-spring-jakarta" \
+    "-DarchetypeArtifactId=spring-boot3" \
```
Comment (MatKuhr) [inline docs-java_versioned_docs/version-v5/getting-started.mdx:45]:
```suggestion
    "-DarchetypeArtifactId=spring-boot-3" \
```

right?

File: docs-java_versioned_docs/version-v5/getting-started.mdx [hunk 2]
```diff
@@ -276,13 +194,7 @@ In general, the SAP Cloud SDK for Java integrates natively into the [Spring Boot
...
-{ label: 'Spring 5', value: 'spring5', },
-{ label: 'Spring 6', value: 'spring6', }]}>
-
-<TabItem value="spring5">
-
-For a **Spring 5** based project please ensure that the application is annotated to scan for components of the SAP Cloud SDK:
+Please ensure that the application is annotated to scan for components of the SAP Cloud SDK:
 
 ```java
 @ComponentScan({"com.sap.cloud.sdk", <your.own.package>})
```
Comment (MatKuhr) [inline docs-java_versioned_docs/version-v5/getting-started.mdx:200]: this needs the Jakarta annotation instead

File: docs-java_versioned_docs/version-v5/guides/logging-overview.mdx [hunk 3]
```diff
@@ -311,7 +311,7 @@ Mind that a value set via the CLI will be overridden if you re-deploy your appli
 #### Recommendation for Spring Boot (e.g. SAP Cloud Application Programming Model)
 
 It's possible to customize the log levels and formatting of your Spring Boot application, e.g. when using the [SAP Cloud Application Programming Model (CAP)](https://cap.cloud.sap/docs/) framework.
-For this we recommend the best practices as they are used in our [`scp-cf-spring`](https://sap.github.io/cloud-sdk/docs/java/getting-started#generating-a-project-from-a-maven-archetype) Maven archetype.
+For this we recommend the best practices as they are used in our [`spring-boot3`](../getting-started#generating-a-project-from-a-maven-archetype) Maven archetype.
```
Comment (MatKuhr) [inline docs-java_versioned_docs/version-v5/guides/logging-overview.mdx:314]:
```suggestion
For this we recommend the best practices as they are used in our [`spring-boot-3`](../getting-started#generating-a-project-from-a-maven-archetype) Maven archetype.
```

File: docs-java_versioned_docs/version-v5/guides/spring-boot-war-deployment.mdx [hunk 4]
```diff
@@ -27,7 +27,7 @@ The advantage of that modern deployment form is that the application can be star
 ## Target Projects of This Guide
 
 This guide is applicable to all Spring-based Java projects which use the modern deployment of Spring as jar file.
-That comprises in the first place any project using the [new CAP stack](https://cap.cloud.sap/docs/) as well as the [Spring Boot Maven archetype](https://search.maven.org/artifact/com.sap.cloud.sdk.archetypes/scp-cf-spring) of the SAP Cloud SDK.
+That comprises in the first place any project using the [new CAP stack](https://cap.cloud.sap/docs/) as well as the [Spring Boot Maven archetype](https://search.maven.org/artifact/com.sap.cloud.sdk.archetypes/spring-boot3) of the SAP Cloud SDK.
```
Comment (MatKuhr) [inline docs-java_versioned_docs/version-v5/guides/spring-boot-war-deployment.mdx:30]:
```suggestion
That comprises in the first place any project using the [new CAP stack](https://cap.cloud.sap/docs/) as well as the [Spring Boot Maven archetype](https://search.maven.org/artifact/com.sap.cloud.sdk.archetypes/spring-boot-3) of the SAP Cloud SDK.
```

**Review by MatKuhr — COMMENTED** [review]
I think openapi docs may also need an update

File: docs-java_versioned_docs/version-v5/getting-started.mdx (hunk 1 above)
Comment (CharlesDuboisSAP) [reply docs-java_versioned_docs/version-v5/getting-started.mdx:45]: [No :)](https://github.com/SAP/cloud-sdk-java/pull/153#pullrequestreview-1752453011)

File: docs-java_versioned_docs/version-v5/getting-started.mdx (hunk 2 above)
Comment (CharlesDuboisSAP) [reply docs-java_versioned_docs/version-v5/getting-started.mdx:200]: Isn't this the workaround:
```java
@ServletComponentScan({"com.sap.cloud.sdk.cloudplatform.servletjakarta", <your.own.package>})
```
Meaning that applications can go back to:
```java
@ServletComponentScan({"com.sap.cloud.sdk", <your.own.package>})
```

**Review by MatKuhr — APPROVED** [review]

**Pull request merged**